couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_docs_actual_disk_size[0]
+        type: long_gauge
+        label: total docs disk size
+        unit: bytes
+        description: Couch docs total size on disk in bytes
+
+      - name: couch.docs.data.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_docs_data_size[0]
+        type: long_gauge
+        label: docs data size
+        unit: bytes
+        description: Couch docs data size in bytes
+
+      - name: couch.docs.disk.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_docs_disk_size[0]
+        type: long_gauge
+        label: docs disk size
+        unit: bytes
+        description: Couch docs total size in bytes
+
+      - name: couch.docs.fragmentation
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_docs_fragmentation[0]
+        type: long_gauge
+        label: docs fragmentation
+        unit: "%"
+        description: Couch docs fragmentation
+
+      - name: couch.spatial.data.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_spatial_data_size[0]
+        type: long_gauge
+        label: spatial data size
+        unit: bytes
+        description: Size of object data for spatial views
+
+      - name: couch.spatial.disk.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_spatial_disk_size[0]
+        type: long_gauge
+        label: spatial disk size
+        unit: bytes
+        description: Amount of disk space occupied by spatial views
+
+      - name: couch.spatial.ops
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_spatial_ops[0]
+        type: long_gauge
+        label: spatial ops
+        description: Spatial operations
+
+      - name: couch.total.disk.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_total_disk_size[0]
+        type: long_gauge
+        label: total disk size
+        unit: bytes
+        description: Couch total disk size
+
+      - name: couch.views.data.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_data_size[0]
+        type: long_gauge
+        label: data size
+        unit: bytes
+        description: Size of object data for views
+
+      - name: couch_views_disk_size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_disk_size[0]
+        type: long_gauge
+        label: disk size
+        unit: bytes
+        description: Amount of disk space occupied by views
+
+      - name: couch.views.fragmentation
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_fragmentation[0]
+        type: long_gauge
+        label: view framentation
+        unit: "%"
+        description: View fragmentation
+
+      - name: cpu.idle.ms
```

Comment:
The period is 1 second, here are metric values. it is gauge
"cpu_idle_ms":[2420,2290,2240,2310,2230,1970,2160,1430,1810,1590,2060,1610,1790,1810,1730,1820,1620,1940,1690,1860,1760,1680,1820,1490,2160,2060,2340,2110,2180,1780,2420,2090,2170,2320,1870,2270,2160,1260,1930,2440,2460,2270,1830,2270,2310,2400,2050,2260,2020,2270,2240,1440,2070,1560,1780,1770,1770,1740,1960,2350]

Reply:
It will be completely useless in that case. Since our default collection interval is 10 sec, we'd catch only every 10th measurement. Probably better to remove it to avoid causing confusion.

Reply:
I will remove.

Reply:
Seems we'd drop too many of metrics in this case... Let's keep it, but properly document why it is no good.